### ISSUE TITLE
StartRequest

### ISSUE THREAD

**Issue by WagZol** [WagZol/u3eBattleship#7]
{
"playerName": "Laci",
"pvp": "true",
}

Nagyobb több ötletem nincs ehhez, igazából a start requestnek szerintem ennyi pont elég. A PVP-vel ki lehet választani hogy player vs player vagy player vs computer legyen

**Comment by bimlas** [comment]
Egyetértek, itt más nem kell.

**Comment by WagZol** [comment]
Esetleg későbbi beállítások kerülhetnek ide. Pl ha meglesz a player lehetősége a hajók formáját megadni egy menüben, vagy hogy mekkora legyen a tábla, stb stb...

**Comment by bimlas** [comment]
Ja, bakkfitty, tényleg, a hajók listáját (formáját) már itt el kéne küldeni, mert az a játék minden fázisában kell: a lerakásnál azt mutatja, melyik a soron következő hajó (vagy te választod ki rákattintva), a játék közben meg lehetne használni arra, hogy mutassa, melyik van még életben.

Bocs, most nézem, hogy ez a request, a #6 response-ban küldöd el a hajók listáját.

**Comment by WagZol** [comment]
Kész a StartRequest, otthon tesztelem!

Amennyiben te elküldted a StartRequest-et és a PVPMode=FALSE úgy elmenti a neved, csinál neked egy táblát és létrehoz egy gépi játékost akinek Computer nevet ad, majd neki is létrehoz egy táblát és lepakoltatja vele a hajókat. Ha egymás után nem tudja 10 alkalom után sem letenni egy Exceptiont fogok neked dobni ilyen formában(Spring 5 én így szeretlek!!!!:D

```
{
    "timestamp": "2018-01-28T19:48:10.471+0000",
    "status": 408,
    "error": "Request timeout",
    "message": "Board is too small",
    "path": "/start"
}
```
Az error nem 1000% hogy ez a szüveg lesz azt tesztelni kell. A lényeg ezt azért adom át neked mert igazából ha belegondolsz neked kéne a kliens oldalon tudni beállítani a tábla méretet, és igazából a hajók formáját is egy settingsben. Ha ez hibás hibaüzenetet kapsz hogy állítsd fentebb. Egyenlőre nem vettük bele a StartRequest-be de szerintem szükség lesz rá

**Comment by WagZol** [comment]
Hasonló exception-ök hibás json esetén is mehetnek szerintem, majd még kitalálom. Így le tudom védeni hogy véletlen se tudj ökörséget küldeni mert visszapattan mielőtt igazi galibát okoz a kódban;)
